Dockerfile

```diff
@@ -1,28 +1,32 @@
 # Client
-FROM public.ecr.aws/bitnami/node:14.17.3 AS client
+FROM registry.access.redhat.com/ubi8/nodejs-14:1 AS client
 
 # Build client
-WORKDIR /client
+# RUN apk add --no-cache git python g++ make
+USER root
```

Comment:
I would leave this out

Reply:
The commented code, right? 
For root user case.... our build script required some additional privileges. But at the end we switch non-root user.

Reply:
I think we should be very deliberate about which commands require root, just as a matter of best practice—like I would enable it for only specific commands vs a batch of them where only one requires it. Feel free to make a case otherwise, I know it's more annoying but I think it creates better clarity around what root is actually required for and prevents accidentally adding commands later with more privilege than they need

Also we should remove the commented-out code lol

Reply:
Currently, build script try to create some dir or file and getting access denied. I checked, those items are part of `WORKDIR`. So, quick solution will be give just write access to non-root user then switch back to non-root user

Reply:
Looks great Pushan thank you!